cmd/dlv/dlv_test.go

```diff
@@ -253,6 +255,57 @@ func TestContinue(t *testing.T) {
 	cmd.Wait()
 }
 
+// TestChildProcessExitWhenNoDebugInfo verifies that the child process exits when dlv launch the binary without debug info
+func TestChildProcessExitWhenNoDebugInfo(t *testing.T) {
+	if _, err := exec.LookPath("ps"); err != nil {
+		t.Skip("test skipped, `ps` not found")
+	}
+
+	dlvbin, tmpdir := getDlvBin(t)
+	defer os.RemoveAll(tmpdir)
+
+	// Make a (good enough) random temporary file name
```

Comment:
Instead of copying this here call BuildFixture passing the LinkStrip build flag. You will have to change TestMain to say `os.Exit(support.RunTestsWithFixtures(m))`

Reply:
OK, I will finish it tomorrow.